client-course-schedulizer/src/utilities/helpers/readCSV.ts

```diff
@@ -132,6 +126,21 @@ export const csvStringToSchedule = (csvString: string): Schedule => {
 };
 
 export const insertSectionCourse = (schedule: Schedule, section: Section, course: Course) => {
+  const { meetings } = section;
+
+  // Check if any meetings are empty, and should be removed
+  let i = 0;
+  let numRemoved = 0;
+  const meetingsCopy = meetings;
+  meetings.forEach((meeting) => {
+    if (meeting.days === [] || meeting.duration === 0) {
+      meetingsCopy.splice(i - numRemoved, 1);
```

Comment:
I think it would be more clear to do something like
```
const meetingIndex = meetingsCopy.indexOf(meeting);
meetingsCopy.splice(meetingsCopy.splice(meetingIndex, 1);
```
rather than keeping track of numRemoved, but I think that's just my personal preference and I'd be fine keeping it as is if you disagree.

Reply:
Also, I think this change may make it so you don't need to make a copy, but I'm not too sure.

Reply:
325d1a9 and 172cce3 should make the logic here a lot simpler (don't know why I didn't think to use `filter()` before).